File: FIL.Contracts/Models/Blogs/BlogResponseModel.cs
using System;
using System.Collections.Generic;

public class guid
{
    public string rendered { get; set; }

    public static object NewGuid()
    {
        throw new NotImplementedException();
    }
}

public class Title
{
    public string rendered { get; set; }
}

public class Content
{
    public string rendered { get; set; }
    public bool Protected { get; set; }
}

public class Excerpt
{
    public string rendered { get; set; }
    public bool Protected { get; set; }
}

public class Meta
{
    public string advanced_seo_description { get; set; }
    public string amp_status { get; set; }
    public string spay_email { get; set; }
    public string jetpack_publicize_message { get; set; }
}

public class Self
{
    public string href { get; set; }
}

public class Collection
{
    public string href { get; set; }
}

public class About
{
    public string href { get; set; }
}

public class Author
{
    public bool embeddable { get; set; }
    public string href { get; set; }
}

public class Reply
{
    public bool embeddable { get; set; }
    public string href { get; set; }
}

public class WpTerm
{
    public string taxonomy { get; set; }
    public bool embeddable { get; set; }
    public string href { get; set; }
}

public class Cury
{
    public string name { get; set; }
    public string href { get; set; }
    public bool templated { get; set; }
}

public class Links
{
    public IList<Self> self { get; set; }
    public IList<Collection> collection { get; set; }
    public IList<About> about { get; set; }
    public IList<Author> author { get; set; }
    public IList<Reply> replies { get; set; }
    public IList<Cury> curies { get; set; }
}

public class BlogResponseModel
{
    public int id { get; set; }
    public DateTime date { get; set; }
    public DateTime date_gmt { get; set; }
    public guid guid { get; set; }
    public DateTime modified { get; set; }
    public DateTime modified_gmt { get; set; }
    public string slug { get; set; }
    public string status { get; set; }
    public string type { get; set; }
    public string link { get; set; }
    public Title title { get; set; }
    public Content content { get; set; }
    public Excerpt excerpt { get; set; }
    public int author { get; set; }
    public int featured_media { get; set; }
    public string comment_status { get; set; }
    public string ping_status { get; set; }
    public bool sticky { get; set; }
    public string template { get; set; }
    public string format { get; set; }
    public Meta meta { get; set; }
    public IList<int> categories { get; set; }
    public IList<int> tags { get; set; }
    public string jetpack_featured_media_url { get; set; }
    public IList<object> jetpack_publicize_connections { get; set; }
    public string yoast_head { get; set; }
    public bool jetpack_likes_enabled { get; set; }
    public bool jetpack_sharing_enabled { get; set; }
    public string jetpack_shortlink { get; set; }
    public Links _links { get; set; }
    public bool? IsImageUpload { get; set; }
}

File: FIL.Web/ClientApp/models/BlogResponseModel.ts
export interface Guid {
    rendered: string;
}

export interface Title {
    rendered: string;
}

export interface Content {
    rendered: string;
    protected: boolean;
}

export interface Excerpt {
    rendered: string;
    protected: boolean;
}

export interface Meta {
    advanced_seo_description: string;
    amp_status: string;
    spay_email: string;
    jetpack_publicize_message: string;
}

export interface Self {
    href: string;
}

export interface Collection {
    href: string;
}

export interface About {
    href: string;
}

export interface Author {
    embeddable: boolean;
    href: string;
}

export interface Reply {
    embeddable: boolean;
    href: string;
}

export interface VersionHistory {
    count: number;
    href: string;
}

export interface PredecessorVersion {
    id: number;
    href: string;
}

export interface WpFeaturedmedia {
    embeddable: boolean;
    href: string;
}

export interface WpAttachment {
    href: string;
}

export interface WpTerm {
    taxonomy: string;
    embeddable: boolean;
    href: string;
}

export interface Cury {
    name: string;
    href: string;
    templated: boolean;
}

export interface Links {
    self: Self[];
    collection: Collection[];
    about: About[];
    author: Author[];
    replies: Reply[];
}
export default class BlogResponseModelList {
    blogResponseModelList: BlogResponseModel[];
}
export interface BlogResponseModel {
    id: number;
    date: Date;
    date_gmt: Date;
    guid: Guid;
    modified: Date;
    modified_gmt: Date;
    slug: string;
    status: string;
    type: string;
    link: string;
    title: Title;
    content: Content;
    excerpt: Excerpt;
    author: number;
    featured_media: number;
    comment_status: string;
    ping_status: string;
    sticky: boolean;
    template: string;
    format: string;
    meta: Meta;
    categories: number[];
    tags: number[];
    jetpack_featured_media_url: string;
    jetpack_publicize_connections: any[];
    yoast_head: string;
    jetpack_likes_enabled: boolean;
    jetpack_sharing_enabled: boolean;
    jetpack_shortlink: string;
    _links: Links;
}

